pages/linux/modinfo.md

```diff
@@ -0,0 +1,7 @@
+# modinfo
+
+> Extract information about a Linux Kernel module.
+
+- Print value(s) of the specified field(s) instead of all the fields:
```

Comment:
I think we should start with `modinfo {{kernel_module}}` and explain what it does.

and then go to `-F`.

Also, instead of /, use |.

Reply:
Updated the description